cli/wordpress-server-child.ts

```diff
@@ -308,6 +362,20 @@ async function runWpCliCommand(
 	};
 }
 
+/**
+ * Runs a WP-CLI command via the queue to ensure commands are serialized.
+ * Waits for server startup to complete before processing.
+ */
+async function runWpCliCommand( args: string[] ): Promise< WpCliResult > {
+	await Promise.allSettled( [ startingPromise ] );
+
+	if ( ! server ) {
+		throw new Error( `Failed to run WP CLI command because server is not running` );
+	}
+
+	return wpCliQueue.enqueue( args );
+}
+
```

Comment:
I would argue we should either inline this in `WpCliCommandQueue::processNext`, or make it a separate method on `WpCliCommandQueue`